lutris/gui/views/grid.py

```diff
@@ -97,8 +100,12 @@ def on_item_activated(self, _view, _path):
 
     def on_selection_changed(self, _view):
         """Handles selection changes"""
-        selected_items = self.get_selected_item()
-        if selected_items:
+        selected_items = self.get_selected_items()
```

Comment:
No, just pass the list to the signal all the time. Don't make the handlers write special checks for list-vs-not-list.